app/views/user_mailer/authentication.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to "Sign In!", new_session_url(auth_token: @authentication.token) %>
```

Comment:
```suggestion
Here's your sign in link:

<%= link_to "Sign In!", new_session_url(auth_token: @authentication.token) %>

<hr/>
<%= new_session_url(auth_token: @authentication.token) %>
```

Let's also add an actual link in case the hyperlink doesn't show/work. Emails are wonky!

Reply:
Implemented, but with a link element so we can be sure it's clickable if the HTML version gets through.